.git-blame-ignore-revs

```diff
@@ -0,0 +1,40 @@
+
+# DRIVERS-2789
+ca372f0c6a1353742437d65131b37544492692c2
```

Comment:
Recommend using `git log -n 1 --format=format:"%H # %s (%cs)" <hash>` to generate entries for this file. e.g.:

```
ca372f0c6a1353742437d65131b37544492692c2 # Sync with 4b7d7573 (2024-02-26)
634d6399bc166d27f290599d8facf034fa17be91 # Sync with 4b7d7573 (2024-02-26)
405e736abf6884c69669fcc060ef76c44f2e63e7 # Sync with 4b7d7573 (2024-02-26)
...
cea087babea68571fa4068ace6b2d326c3a0a22d # DRIVERS-2789 Convert BSON Corpus Spec to Markdown (#1499) (2024-01-23)
e9ae09da918dc7ca7e5ae12a1af6ca34e9c7854e # DRIVERS-2790 Add codespell checker (#1491) (2024-01-10)
eef59d9d386bb054e0eb59e6bd540a030186f899 # DRIVERS-2789 Use Markdown for Specifications Documentation (#1482) (2024-01-04)
```

Reply:
Used `cat .git-blame-ignore-revs | xargs -n1 git log -n 1 --format=format:"%H # %s (%cs)" $1` to update.